code/WorkInProgress/Hydrothings.dm

```diff
@@ -1,12 +1,8 @@
 //Content related to Operation Fuckable Owls
```

Comment:
seems like most of these refactors here are unrelated to butchering

Reply:
`Fixes some stuff in the owlery file while I was there it was annoying.` Its just removing whitespace and adding missing slashes

Reply:
i would atomize those changes out regardless

Reply:
seems like too small of a change to me to warrant its own PR

Reply:
no change is ever too small :beesleep: